src/tools/illink/src/linker/Linker/MemberActionStore.cs

```diff
@@ -68,6 +70,9 @@ public bool TryGetMethodStubValue (MethodDefinition method, out object? value)
 
 		internal bool TryGetFeatureCheckValue (MethodDefinition method, out bool value)
```

Comment:
How do we guarantee this will be called at least once on a given method before the `CodeRewriterStep`?
If I understand the repro for this problem, the callsites in trimmed assemblies are all removed due to branch removal, so we won't mark them and thus won't call this either. This leaves `copy` assemblies and their references. I don't remember exactly what type of processing we do there - if we end up calling `MarkMethod` then this should be fine.

I guess the reason why this works is:
If the `CoreRewriterStep` gets to process a method, that method had to be marked (otherwise it would have been removed) and thus `MarkMethod` ended up calling this `TryGetFeatureCheckValue` and thus populate the cache within.

I must admit this makes me a bit nervous - it feels pretty fragile. Maybe we should make the link from MarkMethod to this a bit more obvious - currently it goes through GetMethodAction which doesn't "seem" like being related to this that much.